tasks/main.yml

```diff
@@ -88,6 +88,13 @@
     - deploydrupal_npm_theme_build
     - deploydrupal_npm_theme_path is defined
 
+- name: clear caches.
+  shell: "{{ deploydrupal_drush_path }} cache-rebuild"
+  args:
+    chdir: "{{ deploydrupal_core_path }}/sites/{{ deploydrupal_site_name }}"
+  when:
+    - deploydrupal_drush_cache_rebuild
+
 # Drush ^9 uses the Drupal service container. We intentionally clear the Drupal
 # cache before the Drush cache to avoid fatal errors due to stale cache in the
 # service container.
```

Comment:
```suggestion
# Drush ^9 uses the Drupal service container. We intentionally clear the Drupal
# cache before the Drush cache to avoid fatal errors due to stale cache in the
# service container.
- name: clear caches.
  shell: "{{ deploydrupal_drush_path }} cache-rebuild"
  args:
    chdir: "{{ deploydrupal_core_path }}/sites/{{ deploydrupal_site_name }}"
  when:
    - deploydrupal_drush_cache_rebuild

```

I was confused at first and thought this comment was in reference to keeping the drush cache clear. What do you think about moving the comment up?

Reply:
Good call. That probably would have avoided this problem in the first place.